src/main/java/spike/command/FindCommand.java

```diff
@@ -0,0 +1,39 @@
+package spike.command;
+
+import spike.task.Task;
+import spike.task.TaskList;
+
+/**
+ * Finds task based on keyword given.
+ */
+public class FindCommand extends Command {
+    private String keyWord;
+
+    /**
+     * Constructor using keyword string.
+     *
+     * @param keyWord the word used to find tasks
```

Comment:
`keyword` should be one word, so `W` should not be capitalized

Reply:
Thanks for the comments, will fix other issues too.